src/main/java/frc/robot/auton/AutonRoutes.java

```diff
@@ -29,11 +55,34 @@ public class AutonRoutes {
         List.of(new AutonMoveInches(-24), new AutonRotate(135), new AutonMoveInches(36))
     );
 
-    static {
+    public static ArrayDeque<AutonAction> MESS_UP_CENTER_RINGS_FROM_AMP_SIDE;
+
+    public static ArrayDeque<AutonAction> MESS_UP_CENTER_RINGS_FROM_TERMINAL_SIDE;
+
+    private static final ArrayDeque<AutonAction> MESS_UP_CENTER_RINGS_TURN_LEFT = new ArrayDeque<AutonAction>(
+        List.of(new AutonMoveInches(312), new AutonRotate(-90), new AutonMoveInches(252))
+    );
+
+    private static final ArrayDeque<AutonAction> MESS_UP_CENTER_RINGS_TURN_RIGHT = new ArrayDeque<AutonAction>(
+        List.of(new AutonMoveInches(312), new AutonRotate(90), new AutonMoveInches(252))
+    );
+
+    /**Paths will slightly differ based on what team we're on, so these will correct them. */
+    public static void setupCorrectAutonPaths() {
         if (DriverStation.getAlliance().get() == Alliance.Red) {
             BACKUP_TURN_BACKUP = RED_BACKUP_TURN_BACKUP;
+            MESS_UP_CENTER_RINGS_FROM_AMP_SIDE = MESS_UP_CENTER_RINGS_TURN_LEFT;
+            MESS_UP_CENTER_RINGS_FROM_TERMINAL_SIDE = MESS_UP_CENTER_RINGS_TURN_RIGHT;
+            SHOOT_ROTATE_BACK_UP_FROM_TERMINAL_SIDE = SHOOT_ROTATE_BACK_UP_FROM_TERMINAL_SIDE_RED;
+            SHOOT_ROTATE_BACK_UP_FROM_AMP_SIDE = SHOOT_ROTATE_BACK_UP_FROM_AMP_SIDE_RED;
+            System.out.println("Configuring auton routes for RED team");
         } else {
             BACKUP_TURN_BACKUP = BLUE_BACKUP_TURN_BACKUP;
+            MESS_UP_CENTER_RINGS_FROM_AMP_SIDE = MESS_UP_CENTER_RINGS_TURN_RIGHT;
+            MESS_UP_CENTER_RINGS_FROM_TERMINAL_SIDE = MESS_UP_CENTER_RINGS_TURN_LEFT;
+            SHOOT_ROTATE_BACK_UP_FROM_TERMINAL_SIDE = SHOOT_ROTATE_BACK_UP_FROM_TERMINAL_SIDE_BLUE;
+            SHOOT_ROTATE_BACK_UP_FROM_AMP_SIDE = SHOOT_ROTATE_BACK_UP_FROM_AMP_SIDE_BLUE;
+            System.out.println("Configuring auton routes for BLUE team");
         }
     }
```

Comment:
We need to turn different directions based on whether we are blue or red so we need to adjust the auton routes accordingly